SU2_CFD/src/solvers/CNEMOEulerSolver.cpp

```diff
@@ -2148,6 +2148,7 @@ void CNEMOEulerSolver::BC_Outlet(CGeometry *geometry, CSolver **solver_container
 
 }
 
+// NECESSARY?
```

Comment:
Is BC_SUPERSONIC_INLET redundant with BC farfield, or should it remain it's own BC? If so, I can work on making it operational, but wanted to check before investing the time.

Reply:
I think supersonic inlet should remain its own BC.  Particularly for the use of inflow conditions not necessarily at freestream conditions. In my mind, scramjet engine outlet conditions could be modeled as a supersonic inlet.